controllers/workspace/status.go

```diff
@@ -155,7 +155,7 @@ func checkServerStatus(workspace *dw.DevWorkspace) (ok bool, err error) {
 	return ok, nil
 }
 
-func getIdeUrl(exposedEndpoints map[string]v1alpha1.ExposedEndpointList) string {
+func getMainUrl(exposedEndpoints map[string]v1alpha1.ExposedEndpointList) string {
 	for _, endpoints := range exposedEndpoints {
 		for _, endpoint := range endpoints {
 			if endpoint.Attributes.GetString(string(v1alpha1.TypeEndpointAttribute), nil) == "ide" {
```

Comment:
I believe we should change it accordingly.
@amisevsk Do you recall what we agreed on? Attribute `main: true`, or `type: main`?
Also, I'm OK with handling it in a separate PR since it depends on plugin registry in addition

Reply:
No memory of the consensus, if we even reached one :D